components/codeblock/codeblock.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <pre>
+    <code
+      class="d-d-block d-bgc-white d-ws-pre-wrap d-p8 d-my4 d-ba d-bc-black-500 d-bar8 d-fs-100 d-lh-200 d-fc-red-400"
+    >{{ text }}</code>
```

Comment:
Where did we actually get the styling for this? was it specified by design or just copied from the existing one in ubervoice?

Reply:
If this is a core component we need to set the styles for this in dialtone and create a documentation page in dialtone (maybe you were already planning on doing this)

Reply:
> Where did we actually get the styling for this? was it specified by design or just copied from the existing one in ubervoice?

It was just copied from the existing one in ubervoice.

> If this is a core component we need to set the styles for this in dialtone and create a documentation page in dialtone (maybe you were already planning on doing this)

This actually completely slipped my mind. By that, you mean making a new Dialtone class like `d-codeblock` with all these classes baked in right?

Reply:
> By that, you mean making a new Dialtone class like `d-codeblock` with all these classes baked in right?

Yep exactly